app/Services/Articles/StockService.php

```
@@ -0,0 +1,152 @@
+
+        return $movement;
+    }
+
+    private function updateStockQuantity(Stock $stock, StockMouvementType $type, float $quantity): void
+    {
+        match ($type) {
+            StockMouvementType::Entree, StockMouvementType::Transfert => $stock->increment('quantity', $quantity),
+            StockMouvementType::Sortie => $stock->decrement('quantity', $quantity),
+            StockMouvementType::Ajustement => $stock->update(['quantity' => $stock->quantity + $quantity]),
+        };
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'updateStockQuantity'. (undefined)

(UnusedPrivateMethod)